src/gl-matrix/mat4.js

```diff
@@ -1278,42 +1278,18 @@ export function perspective(out, fovy, aspect, near, far) {
   out[7] = 0;
   out[8] = 0;
   out[9] = 0;
-  out[10] = (far + near) * nf;
   out[11] = -1;
   out[12] = 0;
   out[13] = 0;
-  out[14] = (2 * far * near) * nf;
-  out[15] = 0;
-  return out;
-}
-
-/**
- * Generates a perspective projection matrix without far plane
- *
- * @param {mat4} out mat4 frustum matrix will be written into
- * @param {number} fovy Vertical field of view in radians
- * @param {number} aspect Aspect ratio. typically viewport width/height
- * @param {number} near Near bound of the frustum
- * @returns {mat4} out
- */
-export function perspectiveInfinite(out, fovy, aspect, near) {
-  let f = 1.0 / Math.tan(fovy / 2);
-  out[0] = f / aspect;
-  out[1] = 0;
-  out[2] = 0;
-  out[3] = 0;
-  out[4] = 0;
-  out[5] = f;
-  out[6] = 0;
-  out[7] = 0;
-  out[8] = 0;
-  out[9] = 0;
-  out[10] = -1;
-  out[11] = -1;
-  out[12] = 0;
-  out[13] = 0;
-  out[14] = -2 * near;
   out[15] = 0;
+  if (far != null && far !== Infinity) {
```

Comment:
That's also fine. 🙂 If anyone has an issue with it, they can just open an issue.